### ISSUE TITLE
Siri Remote, play pause with the big center button.

### ISSUE THREAD

**Issue by ChristopheBis** [jellyfin/Swiftfin#1229]
Hi,

Is there a way to have the behaviour of the center button of the Siri Remote working like the other apps (play /pause)?
The behaviour is the same for Apple TV, prime, YouTube but swift fin is different and frustrating, I have to press the real play pause button which is weird.

Thank you

**Comment by StarFox2873** [comment]
bump

**Comment by vordenken** [comment]
Yes, please! I started using Swiftfin on tvOS and thought the app was broken because I couldn't pause the video.
I never used the "play/pause" button before. This would match every other video players behaviour on tvOS.

**Comment by OlivierAlbertini** [comment]
Hi,

I did it locally, my children were upset xd. Are you still interested ?

**Comment by JPKribs** [comment]
This item should be resolved as of 1.5 on tvOS. When the overlay is present, selecting the progress bar will pause/play content.
